packages/skill-lesson/schemas/resource.ts

```diff
@@ -7,7 +7,7 @@ export const ResourceSchema = z.object({
   slug: z.string(),
   videoResourceId: z.nullable(z.string().optional()),
   description: z.nullable(z.string()).optional(),
-  body: z.nullable(z.any().array().optional()),
+  body: z.nullable(z.any().array().optional().or(z.string().optional())),
```

Comment:
starter is using `text` type + next-mdx-remote for rendering instead of portable text.

sanity has a new markdown editor plugin (https://github.com/sanity-io/sanity-plugin-markdown) but I haven't looked into it yet.